src/rviz_visual_tools.cpp

```diff
@@ -945,6 +945,23 @@ bool RvizVisualTools::publishLine(const Eigen::Affine3d &point1, const Eigen::Af
   return publishLine( convertPoseToPoint(point1), convertPoseToPoint(point2), color, scale );
 }
 
+bool RvizVisualTools::publishLine(const Eigen::Vector3d &point1, const Eigen::Vector3d &point2,
+                              const rviz_visual_tools::colors color, const rviz_visual_tools::scales scale)
+{
```

Comment:
This is a useful function, but it could be simplified by just using this function twice:
```
geometry_msgs::Point RvizVisualTools::convertPoint(const Eigen::Vector3d &point)
```

https://github.com/mcevoyandy/rviz_visual_tools/blob/development/src/rviz_visual_tools.cpp#L1271